checkout_session.go

```diff
@@ -4,6 +4,58 @@ import (
 	"encoding/json"
 )
 
+// CheckoutSessionCustomerDetailsTaxIdsType is the list of allowed values for type
+// on the tax_ids inside customer_details of a checkout session.
+type CheckoutSessionCustomerDetailsTaxIdsType string
```

Comment:
Any chance this could be `CheckoutSessionCustomerDetailsTaxIDsType`? (And similar for other other names in here.)

Reply:
Good catch! Fixed.